solutionbox/code_free_ml/mltoolbox/code_free_ml/analyze.py

```diff
@@ -316,10 +316,14 @@ def run_local_analysis(output_dir, csv_file_pattern, schema, inverted_features):
   Raises:
     ValueError: on unknown transfrorms/schemas
   """
+  sys.stdout.write('Expanding any file patterns...\n')
```

Comment:
ok, well, 12 secs is 'quick', but in terms of thinking printing 'analyze file' has lag, it's useful to show there is some startup costs.